service/index/index_integration_test.go

```diff
@@ -284,7 +289,8 @@ func TestIndex(t *testing.T) {
 	t.Run("seals", func(t *testing.T) {
 		t.Parallel()
```

Comment:
While keeping it for others?

Reply:
Any subtest which calls `setupIndex` can be parallel. Any subtest of that subtest cannot, for the reasons mentioned in my previous answer.